test-resources/swift/leading_comma/configurations/rules.toml

```diff
@@ -4,6 +4,6 @@ query = """(
 (switch_pattern (pattern (simple_identifier) @switch_case))@switch_pattern
 (#eq? @switch_case "@stale_flag")
 )"""
-replace_node = "switch_patttern"
```

Comment:
The changes logic (below), now disallows wrong rules that reference non-existing variables